lib/vis_milk2/ns-eel2/gen-lex-yacc

```diff
@@ -0,0 +1 @@
+yacc -v -d eel2.y && flex eel2.l
```

Comment:
what this file use for?

Reply:
yacc/lex/bison files are used to generate the parser. They aren't used, not sure if they're of any value.